sionna/mapping.py

```diff
@@ -1001,6 +1003,8 @@ def call(self, inputs):
         # Add a dummy dimension for broadcasting. This is not needed when no
         # is a scalar, but also does not do any harm.
         no = tf.expand_dims(no, axis=-1)
+        # Deal with zero or too small values.
+        no += self._eps
```

Comment:
Though it is only a very small increment, wouldn't it be more elegant to simply clip `no`?
I suggest to use `no = tf.math.maximum(no, self._eps)` to ensure that `no` is always positive without introducing any additional impact on larger values of `no`.

Reply:
Yes, that was also my first solution, but I then realized that the clipping would break differentiability on the demapper. I don't know if that's something you were aiming to preserve here.

Reply:
`tf.math.maximum` does not break differentiability of the demapper and should be handled by TF internally (cf. `tf.nn.relu`).

Reply:
Makes total sense, didn't think about hat.